_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -71,15 +65,15 @@ The intent of this rule is to ensure that the `autocomplete` attribute can be us
 
 #### Passed Example 1
 
-Single autocomplete term.
+This `autocomplete` [attribute value][] is only has the required token, and is valid for an `input` element which has a default type of `text`.
```

Comment:
```suggestion
This `autocomplete` [attribute value][] only has the required token, and is valid for an `input` element which has a default type of `text`.
```

Also, should technically be "`type` [attribute value][]", but it is probably OK to keep it that way in an example description (we may want to unify with other descriptions, e.g. Passed Examples 5 and 6 use the technically correct way…)